recipes-ccsp/ccsp/rdk-ppp-manager.bb

```diff
@@ -8,7 +8,7 @@ DEPENDS = "ccsp-common-library dbus rdk-logger utopia halinterface libunpriv"
 
 require recipes-ccsp/ccsp/ccsp_common.inc
 
-GIT_TAG = "v1.0.0"
+GIT_TAG = "RC1.1.0a"
```

Comment:
I wonder should a release candidate change goes to a feature / sprint branch instead?

Reply:
We are planning to merge it to sprint to do the sprint validation and other tests